app/views/standard/seriesDetails.scala.html

```diff
@@ -53,7 +54,7 @@ <h1 class="govuk-label-wrapper">
                                 Continue
                             </button>
 
-                            <a class="govuk-link" href="@routes.HomepageController.homepage()">Cancel</a>
+                            <a class="govuk-link" href="@routes.HomepageController.homepage()">Cancel transfer</a>
```

Comment:
The button in the prototype is a secondary button rather than a link so this should be: 
`<a class="govuk-button govuk-button--secondary" href="@routes.HomepageController.homepage()">Cancel transfer</a>`